coverage_2d/Point2D coverage_bounds
coverage_2d/Point2D[] locations
---
coverage_2d/Point2D[] centroids
